experimental/perf/Dockerfile

```diff
@@ -33,4 +33,7 @@ ARG WANDB_VERSION=""
 RUN pip install --upgrade pip && \
     if [ -z "$WANDB_VERSION" ]; then pip install wandb -qU; else pip install "wandb==$WANDB_VERSION" -qU; fi
 
+RUN pip install numpy \
+    && pip install soundfile
```

Comment:
we have a wandb extra for all the media requirements maybe it would be better to use it?
it's `wandb[media]`

Reply:
didn't know that, will install wandb[media] instead.